docs/management/connectors/action-types/gen-ai.asciidoc

```diff
@@ -86,4 +86,12 @@ Body::      A JSON payload sent to the OpenAI API URL. For example:
 [[gen-ai-connector-networking-configuration]]
 === Connector networking configuration
 
-Use the <<action-settings, Action configuration settings>> to customize connector networking configurations, such as proxies, certificates, or TLS settings. You can set configurations that apply to all your connectors or use `xpack.actions.customHostSettings` to set per-host configurations.
\ No newline at end of file
+Use the <<action-settings, Action configuration settings>> to customize connector networking configurations, such as proxies, certificates, or TLS settings. You can set configurations that apply to all your connectors or use `xpack.actions.customHostSettings` to set per-host configurations.
+
+[float]
+[[gen-ai-connector-token-dashboard]]
+=== Token usage dashboard
+
+Once you've created a Generative AI connector, you can monitor its token usage using the *Generative AI Token Usage* dashboard. Click the *View OpenAI Usage Dashboard for "_<Name>_" Connector* link to open the dashboard.
```

Comment:
> Click the *View OpenAI Usage Dashboard for "_<Name>_" Connector* link to open the dashboard.

From the screenshots in https://github.com/elastic/kibana/pull/159075 it seems like this link is supposed to exist in the connector page, so that seems like screenshots will be affected in this page too.

Reply:
@lcawl The link only exists _after_ the connector has been created and saved, but the current screenshot on this page is creating a new connector (not revisiting/editing an existing connector). I didn't want to take another mostly-identical screenshot just to illustrate the link.

Reply:
```suggestion
Once you've created a Generative AI connector, you can monitor its token usage using the *Generative AI Token Usage* dashboard. Select the connector in *{stack-manage-app}* > *{connectors-ui}* to view its details, then click the *View OpenAI Usage Dashboard for "_<Name>_" Connector* link to open the dashboard.
```